src/website/src/pages/ec/guidelines/spacing/index.mdx

```diff
@@ -12,53 +12,36 @@ import { Paragraph, RemToPixels } from '@ecl/website-components';
   website.
 </Paragraph>
 
-## Two spacing scales
-
-ECL provides two separate sets of scales:
-
-1. Inner spacing: the spacing within the components
-2. Layout spacing: the spacing between the components (+ other elements outside of a component)
-
-For spacing tokens to be able to conform in the baseline grid and contribute to the vertical rhythm, all sizes must be multiples of <RemToPixels rem="0.25" />px. A vertical rhythm starts from a baseline and it’s the basis for everything from the font sizes, line height and image sizes. The spacing tokens always referring to fixed values and thus they are not responsive. All spacing tokens can be used as values for both margins and paddings.
-
 ## Use of space
 
-Space is used to create hierarchy and visual relationships. By using the space tokens, you can achieve visual grouping and other associations both between and within the components. Along with the typography system, they contribute to an overall visual hierarchy, providing users with a structured way of organising information.
+For spacing tokens to be able to conform in the baseline grid and contribute to the vertical rhythm, all sizes must be multiples of <RemToPixels rem="0.25" />px. A vertical rhythm starts from a baseline and it's the basis for everything from the font sizes, line height and image sizes. The spacing tokens always referring to fixed values and thus they are not responsive. All spacing tokens can be used as values for both margins and paddings.
```

Comment:
I'm not sure the reason why here we use the world "token" to mean our utility classes, not that is necessarily wrong per se, but we only do this here in the spacing and so it suprises me.

Reply:
this should be some remains of the time where we used design tokens. I'll rephrase it